icu4c/source/test/intltest/localebuildertest.cpp

```diff
@@ -112,7 +112,7 @@ void LocaleBuilderTest::TestLocaleBuilder() {
         {"B", "fr-FR-1606nict-u-ca-gregory-x-test", "T",
           "fr-FR-1606nict-u-ca-gregory-x-test",
           "fr_FR_1606NICT@calendar=gregorian;x=test"},
-        {"B", "ab-cde-fghij", "T", "cde-fghij", "cde__FGHIJ"},
+        {"B", "cde-fghij", "T", "cde-fghij", "cde__FGHIJ"},
```

Comment:
Can you revert this change, too?

Reply:
In LocaleBuilder, "ab-cde-fghij" is not a valid tag and will fail as illegal argument.